1-org/envs/shared/essential_contacts.tf

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  gcp_scc_admin         = var.gcp_scc_admin == null ? var.group_org_admins : var.gcp_scc_admin
+  gcp_platform_viewer   = var.gcp_platform_viewer == null ? var.group_org_admins : var.gcp_platform_viewer
+  gcp_security_reviewer = var.gcp_security_reviewer == null ? var.group_org_admins : var.gcp_security_reviewer
+  gcp_network_viewer    = var.gcp_network_viewer == null ? var.group_org_admins : var.gcp_network_viewer
+
+  categories_map = {
```

Comment:
Can we add a comment to docs with these enums?

Reply:
Done